templates/user/Profile.html

```diff
@@ -0,0 +1,137 @@
+
+                    {% if user.specialist is not none %}
+                        <p class="flow-text">
+                            <i class="material-icons">contact_phone</i>
+                            <a href="tel:{{ user.phone_number }}"> {{ user.phone_number }}</a>
+                        </p>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+
+        <div class="col s12">
+        {% if user.specialist is not none %}
+                <p class="center-align">
+                <a class="waves-effect waves-light btn modal-trigger"
+                   href="#add_service_activity_modal">Order service</a></p>
+            {% include "AddServiceActivityModal.html" %}
+        {% endif %}
+        {% if activity|length > 0 %}
+        <p class="flow-text center-align">Activities</p>
+
+            <ul class="collapsible" data-collapsible="accordion">
+            {% for a in activity %}
+                <li>
+                    <div class="collapsible-header">
+                        {% if user.specialist is not none %}
+                            {% if a.confirmed == true %}
+                                <div class="chip right" style="background-color: #00c853">
+                                    Confirmed
+                                </div>
+                            {% else %}
+                                <div class="chip right" style="background-color: #d50000">
+                                    Unconfirmed
+                                </div>
+                            {% endif %}
+                            <p class="flow-text">
+                            Customer : <a href="{{ url_for('user_profile', user_id=a.to_user.id) }}">
+                            {{ a.to_user.first_name }} {{ a.to_user.last_name }}
+                            </a>
+                            </p>
+                        {% else %}
+                            {% if a.confirmed == true %}
+                                <div class="chip right" style="background-color: #00c853">
+                                    Confirmed
+                                </div>
+                            {% else %}
+                                <div class="chip right" style="background-color: #d50000">
+                                    Unconfirmed
+                                </div>
+                            {% endif %}
+                            <p class="flow-text">Specialist : <a href="{{ url_for('user_profile', user_id=a.from_user_id) }}">
+                            {{ a.from_user.first_name }} {{ a.from_user.last_name }}
+                            </a>
+                            </p>
+                        {% endif %}
+
+                    </div>
+                    <div class="collapsible-body">
+                        {% if a.end is not none %}
+                        <p class="flow-text"><i class="material-icons">event</i> {{ a.start }} ||| {{ a.end }}</p>
```

Comment:
You actually need only `{{ a.start }} ||| {{ a.end }}` inside `if`, not the whole `<p class.... <i class`